### PR TITLE
Add aarch/ppc

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
This is ready to be merged @conda-forge/inotify_simple :)

**Comment by Tobias-Fischer** [comment]
Hi @conda-forge/help-osx-arm64 @wolfv any chance of merging this? It's been a few days, and it's needed for https://github.com/conda-forge/boa-feedstock/pull/20

**Comment by isuruf** [comment]
Please add yourself as a maintainer

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 8440ffe49c4ae81a8df57c1ae1eb4b6bfa7acb830099bfb3e305b383005cc128
 
 build:
-  number: 1
+  number: 2
```
Comment (isuruf) [inline recipe/meta.yaml:14]: Keep this as 1

**Comment by isuruf** [comment]
Can you make this recipe noarch: python?

**Comment by Tobias-Fischer** [comment]
This package doesn't work for Win - can it still be noarch: python?

**Comment by isuruf** [comment]
> This package doesn't work for Win - can it still be noarch: python?

Sure. Add `__unix` to `run` requirements.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,24 +10,22 @@ source:
   sha256: 8440ffe49c4ae81a8df57c1ae1eb4b6bfa7acb830099bfb3e305b383005cc128
 
 build:
+  noarch: python
   number: 1
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
  number: 2
```

**Comment by Tobias-Fischer** [comment]
Looks like the changes worked fine, thanks @isuruf

**Pull request merged**